deploy/truefi2.ts

```diff
@@ -96,7 +98,9 @@ deploy({}, (_, config) => {
   // New bare contracts
   const trueFiPool2 = contract(TrueFiPool2)
   const implementationReference = contract(ImplementationReference, [trueFiPool2])
+  const chainlinkTruTusdOracle = contract(ChainlinkTruTusdOracle)
   const chainlinkTruUsdcOracle = contract(ChainlinkTruUsdcOracle)
+  const chainlinkTruUsdtOracle = contract(ChainlinkTruUsdtOracle)
```

Comment:
You should name them differently, otherwise 2nd and 3rd deploys will be skipped

Reply:
Aren't these 3 different contracts?

Reply:
They are, missed the parallel PR. No need to change then

Reply:
You do raise a good readability point. With this naming scheme it would be easy for us to overlook a typo.

Doesn't have to be fixed now, but something to keep in mind for the future.